doorkeeper.gemspec

```diff
@@ -31,8 +31,8 @@ Gem::Specification.new do |gem|
     "funding_uri" => "https://opencollective.com/doorkeeper-gem",
   }
 
-  gem.add_dependency "railties", ">= 5"
```

Comment:
This can break a lot of applications. I believe we should set such requirements only if we're using something version-specific. So I'm for ignoring EOL rubies/rails in our CI, but against strict dependencies versions when we really don't need them

Reply:
OK, I see.
I will cancel this  gem.add_dependency  change.